setup.cfg

```
@@ -47,6 +47,10 @@ dinghy =
     graphql/*.*
     templates/*.*
 
+[options.entry_points]
+console_scripts =
+    dinghy = dinghy.__main__
```

```suggestion
    dinghy = dinghy.__main__:cli
```
Did it work for you without this?  I had to add this.

---

Yea it worked for me without that, weird.